db/types/base.py

```diff
@@ -131,10 +146,22 @@ def _add_to_db_type_hintsets(db_types, hints):
             else:
                 db_types_hinted[db_type] = tuple(hints)
 
+    # all types get the "any" hint
     all_db_types = known_db_types
     hints_for_all_db_types = (hints.any,)
     _add_to_db_type_hintsets(all_db_types, hints_for_all_db_types)
 
+    # string-like types get the "string_like" hint
+    string_like_db_types = (
+        PostgresType.CHARACTER_VARYING,
+        PostgresType.CHARACTER,
+        PostgresType.TEXT,
+        MathesarCustomType.URI,
```

Comment:
This sort of info is also in the `cast.py` file.  Did you consider trying to unify these with those definitions?  I ask because it makes me nervous to have this defined in more than one place.

Reply:
I didn't notice the duplication. I'll investigate.